scripts/check_license.sh

```diff
@@ -62,7 +68,11 @@ for FILE in $VERSIONED_GO_FILES; do
             RETURN_VALUE=1
 
             if ! $VERBOSE; then
-                if $INPLACE; then
+                if $UPGRADE; then
+                    sed -i.orig s/Copyright\ \(C\)\ 2019-....\ Algorand,\ Inc\./Copyright\ \(C\)\ 2019-$CURRENT_YEAR\ Algorand,\ Inc./ "$PROJECT_ROOT/$FILE" && \
```

Comment:
1. maybe replace `....` with PREV_YEAR  to be exact?
2. if I recall right `sed -i.orig` is mac only syntax (btw, `sed -i '' -e` also works without creating extra files). I think this cmd should be BSD/GNU sed universal.

Reply:
I'd keep it general to catch missing years, but don't feel strongly on it.